.github/workflows/ci.yml

```diff
@@ -0,0 +1,59 @@
+name: continuous-integration
```

Comment:
You can drop this file, it's duplicating our own workflow https://github.com/ProtonMail/vobject/blob/master/.github/workflows/actions.yml

Reply:
got it